Sources/WebAuthn/Ceremonies/Shared/COSE/COSEAlgorithmIdentifier.swift

```diff
@@ -17,41 +17,41 @@ import Crypto
 /// COSEAlgorithmIdentifier From §5.10.5. A number identifying a cryptographic algorithm. The algorithm
 /// identifiers SHOULD be values registered in the IANA COSE Algorithms registry
 /// [https://www.w3.org/TR/webauthn/#biblio-iana-cose-algs-reg], for instance, -7 for "ES256" and -257 for "RS256".
-public enum COSEAlgorithmIdentifier: Int, RawRepresentable, CaseIterable, Encodable, Sendable {
+public enum COSEAlgorithmIdentifier: Int, RawRepresentable, CaseIterable, Codable, Sendable {
```

Comment:
Side note, we should get rid of this before 1.0.0

https://github.com/swift-server/swift-webauthn/issues/91